superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts

```diff
@@ -26,10 +26,6 @@ describe('AdhocFilters', () => {
     cy.verifySliceSuccess({ waitAlias: '@postJson' });
   });
 
-  xit('Should not load mathjs when not needed', () => {
-    cy.get('script[src*="mathjs"]').should('have.length', 0);
-  });
-
```

Comment:
Note to other reviewers: we're fully removing `mathjs` as a dependency right now due to bundle size and compatibility problems with Webpack 5. Related PR removing `mathjs` from `superset-ui`: https://github.com/apache-superset/superset-ui/pull/1362